ci-operator/step-registry/openstack/test/manifests/openstack-test-manifests-commands.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+export OS_CLIENT_CONFIG_FILE="${CLUSTER_PROFILE_DIR}/clouds.yaml"
+
+# The script needs yq
+export PATH="${PATH}:/usr/local/bin"
```

Comment:
Consider updating the path in the Dockerfile instead, https://github.com/openshift/installer/blob/b852762/images/openstack/Dockerfile.ci#L44

Reply:
OK, done in https://github.com/openshift/installer/pull/4459

This will now fail until that one is merged.